File: Questions solution/remove-element.c
#include <stdio.h>

int main() {
  int nums[] = {3, 2, 2, 3, 6};
  int expectedNums[5];

  for (int i = 0; i < sizeof(nums) / sizeof(nums[0]); i++) {
    for (int j = 0; j < 3; j++) {
      if (nums[i] != 3) {
        expectedNums[j] = nums[i];
      }
    }
  }

  for (int i = 0; i < 3; i++) {
    printf("%d", expectedNums[i]);
  }

  return 0;
}

File: remove-element.py
class Solution:
    def removeElement(self, nums: list[int], val: int) -> int:
        expectedNums: int = 0

        for num in nums:
            if num != val:
                expectedNums += 1

        return expectedNums


practice = Solution()

nums: list[int] = [3,2,2,3]
print(practice.removeElement(nums, 3))